.golangci.goheader.template

```diff
@@ -0,0 +1,13 @@
+Copyright © {{ YEAR }} Meroxa, Inc.
```

Comment:
I think this can be Meroxa, Inc. and GophersLab, but we can double check with Ali.